sys-boot/refind/refind-0.11.4.ebuild

```diff
@@ -49,11 +50,11 @@ src_prepare() {
 
 	# bug 598647 - PIE not supported
 	sed -e '/^CFLAGS/s:$: -fno-PIE:' -i Make.common || die
-
+	LIB_DIR="$(get_libdir)" || die
```

Comment:
`get_libdir` is a built-in function and doesn't need to die.

Reply:
That was more to fail if it could not get libdir